docs/DeveloperGuide.adoc

```diff
@@ -331,32 +331,146 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: UC01 - Add an order
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User key in the order details.
+2.  Delino add the order details.
+3.  Delino displays order added.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. Delino detects invalid syntax.
+[none]
+** 1a1. Delino shows an error message.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+[discrete]
+=== Use case: UC02 - Clear all orders
+
+*MSS*
+
+1.  User requests to clear all orders.
+2.  Delino clear all existing orders.
+3.  Delino displays order cleared message.
 +
+Use case ends.
+
+*Extensions*
+
 [none]
-** 3a1. AddressBook shows an error message.
+* 1a. Delino detects invalid syntax.
+[none]
+** 1a1. Delino shows an error message.
 +
-Use case resumes at step 2.
+Use case ends.
 
-_{More to be added}_
+[none]
+* 1b. Delino detects no orders.
+[none]
+** 1b1. Delino shows no order to be cleared message.
++
+Use case ends.
+
+[discrete]
+=== Use case: UC03 - Delete an order
+
+*MSS*
+
+1.  User requests to list orders (UC10).
+2.  User requests to delete a specific order in the list.
+3.  Delino deletes the order.
```

Comment:
Similar to my review on the deleting all the orders, maybe Delino can ask the user if he or she really wants to delete the orde before actually deleting the order.